kibbeh/src/lib/createChatMessage.ts

```diff
@@ -18,7 +18,7 @@ export const createChatMessage = (
   const testAndPushToken = (item: string) => {
     const isLink = linkRegex.test(item);
     const withoutAt = item.replace(/@|#/g, "");
-    const isMention = roomUsers.find((m) => withoutAt === m.username);
+    const isMention = roomUsers.find((m) => withoutAt === m.username && item.includes(`@${withoutAt}`));
```

Comment:
The replace line seem to be handling both `@` and `#`, and your includes only looks for an `@`

Reply:
My code only looks for an @ symbol because it only affects when you try to mention someone and not when you whisper to someone. Whisper uses #@

Reply:
If both features are still working, it's ok

Reply:
![image](https://user-images.githubusercontent.com/53215039/116200268-c0695f80-a6fd-11eb-804e-933eeacdedec.png)
Just tested it and it works perfectly!